backend/vxlan/vxlan_windows.go

```diff
@@ -0,0 +1,25 @@
+// +build windows
```

Comment:
Do you need this file since vxlan isn't currently supported on Windows?

Reply:
Merged VXLAN support